docker/install/ubuntu_install_androidsdk.sh

```diff
@@ -68,20 +67,20 @@ extras;google;simulators
 extras;google;webdriver
 extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2
 extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2
-lldb;2.3
```

Comment:
I check with `sdkmanager --list`, I Deleted the package that does not currently exist.